src/main/java/org/mule/extension/ftp/internal/command/FtpListCommand.java

```diff
@@ -157,15 +156,13 @@ private void doList(FileConnectorConfig config,
   }
 
   private Iterator<FTPFile[]> getFtpFileIterator() throws IOException {
-    // Check if MLSD command is supported
-    try {
```

Comment:
Should we keep the try, just in case?

Reply:
Yeah, I guess so, worst case is the server is sending a real error and we try again with the old command for no reason.

Reply:
yeah, I guess so, worst case is the server is sending a real error and we are trying again for no reason.